.github/workflows/release.yml

```
@@ -0,0 +1,57 @@
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)